src/tactic/ui/panel/layout_wdg.py

```diff
@@ -4429,8 +4429,8 @@ def __init__(my, **kwargs):
         default_element_names = config.get_element_names()
 
 
-        views = ['table', 'tile', 'list', 'content', 'navigate', 'schedule', 'checkin', 'tool', 'browser', 'card', 'overview']
-        labels = ['Table', 'Tile', 'List', 'Content', 'Navigator', 'Task Schedule', 'Check-in', 'Tools', 'File Browser', 'Card', 'Overview']
+        views = ['table', 'tile', 'list', 'content', 'navigate', 'schedule', 'checkin', 'tool', 'browser', 'card', 'collection', 'overview']
+        labels = ['Table', 'Tile', 'List', 'Content', 'Navigator', 'Task Schedule', 'Check-in', 'Tools', 'File Browser', 'Card', 'Collections', 'Overview']
```

Comment:
Shoud it be?  It's like Tools ... it doesn't show one collection but many collections.